src/app/ngrx/reducers/episode-metrics.reducer.ts

```diff
@@ -1,68 +1,83 @@
 import * as ACTIONS from '../actions';
 
 export interface EpisodeMetricsModel {
-  seriesId: number;
-  id: number;
+  podcastId: string;
   guid?: string;
   page?: number;
   monthlyReach?: any[][];
   weeklyReach?: any[][];
   dailyReach?: any[][];
   hourlyReach?: any[][];
+  charted?: boolean;
   loaded?: boolean;
   loading?: boolean;
   error?: any;
 }
 
 const initialState = [];
```

Comment:
The shape of this reducer's state is an array of `EpisodeMetricsModel`s. Some of our reducers that were written later use the entity pattern. If I had infinite time, I would convert all the collection reducers to use @ngrx/entity and its library of helpers for managing and selecting state.